iohk/NixOps.hs

```diff
@@ -525,17 +534,30 @@ scpFromNode o c (fromNodeName -> node) from to = do
     ExitFailure code -> TIO.putStrLn $ "scp from " <> node <> " failed with " <> showT code
 
 sshForEach :: Options -> NixopsConfig -> [Text] -> IO ()
-sshForEach o c cmd = 
+sshForEach o c cmd =
   nixops o c "ssh-for-each" ("--": cmd)
 
+deployed'commit :: Options -> NixopsConfig -> NodeName -> IO ()
```

Comment:
@domenkozar, basically this came up when Philipp was working with the cluster and he doubted that the actually running version was what he wrote in `cardano-sl-src.json`.

This is not a hack, I believe -- this is a _sensor_, a debug tool to figure out the actual reality of what was deployed on a particular cluster.